StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkVerification/NetworkingLinkVerificationDataSource.swift

```diff
@@ -51,7 +51,8 @@ final class NetworkingLinkVerificationDataSourceImplementation: NetworkingLinkVe
             consumerSession: nil,
             apiClient: apiClient,
             clientSecret: clientSecret,
-            analyticsClient: analyticsClient
+            analyticsClient: analyticsClient,
+            isTestmode: manifest.isTestMode
```

Comment:
```
isTestmode: manifest.isTestMode
```

one uses lowercase "m" (isTestmode) the other uses upper case "M" (manifest.isTestMode) - is that expected?